web/main/templates/includes/action_buttons.html

```diff
@@ -48,15 +48,23 @@
     </form>
       {% endif %}
   {% endif %}
+    
   {% if exportable %}
     <a class="action one-line export export-{{ section.is_annotated|yesno:"has,no" }}-annotations" role="button" rel="nofollow" href="#">Export</a>
+
     {% if not resource and not section and request.user.is_superuser and not FORCE_EXPERIMENTAL_EXPORT %}
-      <a class="action one-line export export-experimental export-{{ section.is_annotated|yesno:"has,no" }}-annotations" role="button" rel="nofollow" href="#">Experimental Export</a>
+      <a class="action one-line export export-experimental export-{{ section.is_annotated|yesno:"has,no" }}-annotations" role="button" rel="nofollow" href="#">DocXport (TOC Only)</a>
```

Comment:
Renamed the python-docx export to `DocXport (TOC Only)` since there are a few experiments going.